gnomad/sample_qc/ancestry.py

```diff
@@ -259,52 +259,60 @@ def assign_population_pcs(
 def run_pca_with_relateds(
     qc_mt: hl.MatrixTable,
     related_samples_to_drop: Optional[hl.Table],
+    outlier_samples_to_drop: Optional[hl.Table],
```

Comment:
One thought is to modify this to additional samples to drop or something like that. Right now for removing unreleasable samples we just combine them with the related sample HT that we pass to this function, but we could just pass that table here too.

Reply:
I like additional samples to drop. Don't we drop the unreleasable samples and non-high-quality samples completely before running this step?

Reply:
Nope https://github.com/broadinstitute/gnomad_qc/blob/a828b1529a439fc41ecc73647a442c7e63897701/gnomad_qc/v3/sample_qc/sample_qc.py#L457

```
    if not include_unreleasable_samples:
        logger.info("Excluding unreleasable samples for PCA.")
        samples_to_drop = samples_to_drop.union(
            qc_mt.filter_cols(
                ~project_meta.ht()[qc_mt.col_key].releasable
                | project_meta.ht()[qc_mt.col_key].exclude
            )
            .cols()
            .select()
        )
    else:
        logger.info("Including unreleasable samples for PCA")

    return run_pca_with_relateds(qc_mt, samples_to_drop, n_pcs=n_pcs)
```

We do drop samples that fail hard filtering

Reply:
ah ok, for the subpop analysis the unreleasable samples are dropped

Reply:
Yeah, I think that's fine